sprite.go

```diff
@@ -362,59 +416,53 @@ func (p *Sprite) OnCloned__1(onCloned func()) {
 	})
 }
 
+func (p *Sprite) fireTouchStart(obj *Sprite) {
+	if p.hasOnTouchStart {
+		p.doWhenTouchStart(p, obj)
+	}
+}
+
 func (p *Sprite) fireTouched(obj *Sprite) {
```

Comment:
We should rename `touched` related fields or methods (`fireTouched`, `hasOnTouched`, ...) to `touching` to keep consistent with conclusion [here](https://github.com/goplus/spx/pull/340#discussion_r1780594928)?

Reply:
ok